redis/tests/support/cluster.rs

```diff
@@ -216,8 +218,10 @@ impl RedisCluster {
                                     out.read_to_string(&mut str).unwrap();
                                     str
                                 });
+
+                                let log_file_contents = std::fs::read_to_string(log_file_path).unwrap();
```

Comment:
How verbose are these log files potentially? I get nervous any time I see a file get read into a string :sweat_smile:

Reply:
so far I've only seen 4 lines of text in a file. Remember that these are read only if the server fails initialization, not after the server has started running, so I assume it won't have a chance to grow.